public/chat_header_button.tsx

```diff
@@ -154,16 +155,29 @@ export const HeaderChatButton: React.FC<HeaderChatButtonProps> = (props) => {
           onKeyPress={onKeyPress}
           onKeyUp={onKeyUp}
           prepend={
-            <EuiIcon type={chatIcon} size="l" onClick={() => setFlyoutVisible(!flyoutVisible)} />
+            <EuiIcon
+              aria-label="toggle chat flyout icon"
+              type={chatIcon}
+              size="l"
+              onClick={() => setFlyoutVisible(!flyoutVisible)}
+            />
           }
           append={
             <span className="llm-chat-header-shortcut">
               {inputFocus ? (
-                <EuiBadge className="llm-chat-header-shortcut-enter" color="hollow">
+                <EuiBadge
+                  title="press enter to chat"
+                  className="llm-chat-header-shortcut-enter"
+                  color="hollow"
+                >
                   ⏎
                 </EuiBadge>
               ) : (
-                <EuiBadge className="llm-chat-header-shortcut-cmd" color="hollow">
+                <EuiBadge
+                  title="press ⌘ + / to start typing"
```

Comment:
Don't know if the text will confuse Windows user.

Reply:
Yes, we have a task to improve this